addons/mail/static/src/models/message/message.js

```diff
@@ -625,6 +640,12 @@ function factory(dependencies) {
         guestAuthor: many2one('mail.guest', {
             inverse: 'authoredMessages',
         }),
+        /**
+         * States whether the message has one or more attachments.
```

Comment:
I have updated it but I don't like it since "some attachments" doesn't seam include "one attachment". "one or more" leave 0 ambiguity.
But yeah, that is also nit-picking.